runtime/oti/j9cfg_builder.h

```diff
@@ -92,14 +92,23 @@
 #define J9_JIT_DATA_CACHE_SIZE (8 * 1024 * 1024)
 #endif /* J9VM_ARCH_X86 && !J9VM_ENV_DATA64 */
 
-#if defined(J9ZOS390) && defined(J9VM_ENV_DATA64)
+
+#define J9_OS_STACK_GUARD (16 * 1024)
+
+#if defined(J9VM_ENV_DATA64) && defined(J9ZOS390)
 /* Use a 1MB OS stack on z/OS 64-bit as this is what the OS
  * allocates anyway, using IARV64 GETSTOR to allocate a segment.
  */
 #define J9_OS_STACK_SIZE (1024 * 1024)
-#else /* J9ZOS390 && J9VM_ENV_DATA64 */
+#elif defined(J9VM_ENV_DATA64) && defined(AIXPPC)
```

Comment:
Do we want to do this for Linux as well? The VRs are saved there as well.

`J9VM_ARCH_POWER` would be the define to include both.

Reply:
If so, comment below should also be adjusted.